docs/deployments/predictors.md

```diff
@@ -310,6 +332,17 @@ class ONNXPredictor:
             Prediction or a batch of predictions.
         """
         pass
+
+    def post_predict(self, response, payload, query_params, headers):
+        """(Optional) Called after the response has been sent back. Useful for sending metrics, updating in-memory cache, or anything that doesn't have to be on the critical path.
```

Comment:
`Called after the response has been sent back. Useful for sending metrics, updating in-memory cache, or anything that doesn't have to be on the critical path.` -> `Called in the background after returning a response. Useful for tasks that the client doesn't need to wait on before receiving a response such as recording metrics or storing results.` I think this description is a little longer but also easier to understand. Let me know what you think.

Reply:
Yep, I like how this sounds.